src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/ExceptionRecordTest.java

```diff
@@ -1,15 +1,10 @@
 package uk.gov.hmcts.reform.bulkscan.endtoendtests;
 
 import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.bulkscan.endtoendtests.dsl.Await;
```

Comment:
that's not dsl, helper perhaps, but not dsl. just executing methods which assert/poll/verify stuff. maybe `api` (following assertj methodology

Reply:
moved to helpers